bee-ledger/src/workers/snapshot/download.rs

```diff
@@ -40,7 +40,7 @@ async fn download_snapshot_header(download_url: &str) -> Result<SnapshotHeader,
                 }
             }
         }
-        Err(e) => warn!("Downloading snapshot header failed: {:?}.", e.to_string()),
+        Err(e) => debug!("Downloading snapshot header failed: {:?}.", e.to_string()),
```

Comment:
Actually wondering, should this be a warning for full and debug for delta ? Or do we just want to discover at the end that no source were available/working ?

Reply:
I would expect most users to only care about the node starting correctly. So checking at the end should suffice.